homepage/templates/homepage/app_layout.html

```diff
@@ -48,6 +53,16 @@
   </div>  
 </nav>
 
+
+  {% if messages %}
+  <ul class="messages">
+      {% for message in messages %}
+      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>

```

Comment:
please check that in case of message.tags = None we don't get empty list item like:

-

Reply:
> please check that in case of message.tags = None we don't get empty list item like:

Can you please elaborate regarding this ? I'm not sure i understand.

From what i see in the django message documentation, each message must have a tag, and the tag represents the message level and also some additional information about the message if needed.
Also - i dont think if message.tags = None then we will get an empty list displayed (i could not reproduce this)
https://docs.djangoproject.com/en/3.2/ref/contrib/messages/#message-tags

Also - how do i need to change the code above to check the case of message.tag == None?
right now i see there is a check `{% if message.tags %}`

Reply:
opss...
I missed the {{ message }} at the end of the line